apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -108,36 +129,31 @@ export function InstructorAssessmentSettings({
                   <input
                     type="text"
                     class="form-control"
+                    res.locals.course.path,
                     id="module"
                     name="module"
-                    value="${resLocals.assessment_module
-                      ? resLocals.assessment_module.heading
-                      : ''}"
-                    disabled
+                    list="modulesList"
```

Comment:
Can you say more about why you're using a datalist instead of a `<select>`, both here and for the set? AFAIK the only benefit is that one can add a custom value that isn't already in the list, but I'm actually not sure if we want that here. That is, I might prefer to force folks to manage that via the course settings pages.

Reply:
Usage of a datalist here was suggested [here](https://github.com/PrairieLearn/PrairieLearn/pull/10653#discussion_r1759210972) so I was trying to incorporate that. I do agree with your strategy of managing sets and modules from the course settings pages so I have updated to use a `<select>` in [9173a63](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/9173a63860cec149d3796686882b0ec384a3dc16).